cmd/root.go

```diff
@@ -27,21 +41,19 @@ var root = &cobra.Command{
 	Long: `Serverless functions
 
 Create, build and deploy functions in serverless containers for multiple runtimes on Knative`,
-	Example: `
-# Create a node function called "node-sample" and enter the directory
-kn func create myfunc && cd myfunc
-
-# Build the container image, push it to a registry and deploy it to the connected Knative cluster
-# (replace <registry/user> with something like quay.io/user with an account that have you access to)
-kn func deploy --registry <registry/user>
+	Example: replaceNameInTemplate("func", "example"),
+}
 
-# Curl the service with the service URL
-curl $(kn service describe myfunc -o url)
-`,
+func replaceNameInTemplate(name, template string) string {
+	var buffer bytes.Buffer
+	exampleTemplate.ExecuteTemplate(&buffer, template, name)
```

Comment:
CI is failing here with
> cmd/root.go:49:33: Error return value of `exampleTemplate.ExecuteTemplate` is not checked (errcheck)
	exampleTemplate.ExecuteTemplate(&buffer, template, name)
Can you check for the error condition here?

I suppose `replaceNameInTemplate()` should return `string, err` and I guess that means that `NewRootCmd()` should return `*cobra.Command, err` as well, which ultimately bubbles up to `plugin.go`.  :thinking: